backend/persistent/datastores/Datastores.js

```diff
@@ -5,6 +5,7 @@ import * as ApacheDrill from './ApacheDrill';
 import * as IbmDb2 from './ibmdb2';
 import * as ApacheLivy from './livy';
 import * as ApacheImpala from './impala';
+import * as DataWorld from './DataWorld';
```

Comment:
Most of the project uses lowercase filenames and I'm planning to get rid of the exceptions.
Would you make this file lowercase?